### DISCUSSION TITLE
Info about queue_jobs configuration option

### DISCUSSION THREAD

**Discussion by illambo** [getsentry/sentry-laravel#831] [Q&A] [answered]
Hi, about [queue_jobs ](https://github.com/getsentry/sentry-laravel/blob/debe205d0f9ba4058312a55228ab40401b286e79/config/sentry.php#L70C15-L70C15) option I don't understand _"Capture queue jobs as spans when executed on the sync driver"_, I don't think the filter for the "sync" driver is applied, maybe I misunderstood?
Thanks

**Comment by stayallive** [comment]
> I don't think the filter for the "sync" driver is applied, maybe I misunderstood?

What do you mean exactly? I'm afraid I don't know what you are asking 😅 

Can you clarify what exactly isn't working?

**Reply by illambo** [reply]
Hi sure (sorry)!  It seems to me that the jobs queued even in the non-sync queue are still tracked (or at least that's how it seems to me if I look at the sentry dashboard, the option _queue_job_transactions_ is set to false). 
Have I made it clear to you? I thought that with this option it would only track the jobs executed on the queue with driver sync.

**Reply by stayallive** [reply, marked as answer]
Ah yes. I understand now. This is indeed a bug that is going to be solved with #820, we expect to release this early next week. The currently solution is to also disable the `queue_jobs` option so no jobs are tracked.

**Reply by illambo** [reply]
Ok, thanks!
